x-pack/plugins/uptime/public/pages/monitor.tsx

```diff
@@ -32,7 +46,8 @@ export const MonitorPage: React.FC = () => {
   useTrackPageview({ app: 'uptime', path: 'monitor' });
   useTrackPageview({ app: 'uptime', path: 'monitor', delay: 15000 });
 
-  const nameOrId = selectedMonitor?.monitor?.name || monitorId || '';
+  const nameOrId =
+    selectedMonitor?.monitor?.name || getIdToDisplay(monitorId, selectedMonitor) || '';
```

Comment:
```suggestion
    selectedMonitor?.monitor?.name || getIdToDisplay(monitorId, selectedMonitor);
```

I think `getIdToDisplay` always returns a `string`; if that's right, we don't need this last `||`.